webapp/channels/webpack.config.js

```diff
@@ -194,6 +196,7 @@ var config = {
                 {from: 'src/images/purchase_alert.png', to: 'images'},
                 {from: '../node_modules/pdfjs-dist/cmaps', to: 'cmaps'},
                 {from: 'src/components/initial_loading_screen/initial_loading_screen.css', to: 'css'},
+                {from: 'src/workers/sw.js', to: ''},
```

Comment:
Why not have Webpack compile the service worker? Is that just for this initial version?

Reply:
We cannot use webpack to compile because then it generates file hashes for the generated assets. And therefore the file hash changes every time the file contents changes. This breaks the service worker update logic. The browser updates the service worker only if the name remains the same and the contents of the file changes.